src/style.css

```diff
@@ -0,0 +1,94 @@
+body {
+  background-color: gainsboro;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.container {
+  background-color: #fff;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+  box-shadow: 5px 5px #000;
+}
+
+.content {
+  margin-left: 20px;
+  margin-right: 20px;
```

Comment:
Instead of repeating the properties, it would be much better to use it in one line so that your code can be more readable and clean 😃. Check out all your styling and correct it accordingly.

Reply:
Thank you for your review, changes implemented